.github/workflows/capgen_python_linting.yaml

```diff
@@ -0,0 +1,31 @@
+name: Capgen Python Linting
+
+on:
+  workflow_dispatch:
+  pull_request:
+    types: [opened, synchronize, reopened]
+  push:
+    branches:
+      #Trigger workflow on push to any branch or branch heirarchy:
+      - '**'
+
+jobs:
+  capgen_python_linting:
+    strategy:
+      matrix:
+        python-version: [3.7]
+    if: github.event_name == 'pull_request' || github.event_name == 'workflow_dispatch' || github.repository == 'NCAR/ccpp-framework'
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v2
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Install dependencies
+      run: |
+        python3 -m pip install --upgrade pip
+        pip3 install pylint
+    - name: Run pylint against capgen files
+      run: |
+        cd test && pylint
```

Comment:
@mwaxmonsky I know I said "if it's easy, go ahead and throw in a pylint test" but now I'm thinking there's more work to be done on linting all of the python files (or at least those that were updated). As written, this runs pylint on the test files, which is probably not quite sufficient. 

My vote (sorry to make you undo things!) is to remove this and make an issue.

Reply:
Yeah, I was hoping that since there was a `.pylintrc` file already, this would be a fairly quick add but it looks like that file hasn't been touched in a while and there's a few things that need to be examined so I'll remove this from here and make a separate PR.  No worries!